addons/product/models/product_pricelist.py

```diff
@@ -486,23 +485,7 @@ def _get_pricelist_item_name_price(self):
                 item.name = _("All Products")
 
             if item.compute_price == 'fixed':
-                decimal_places = self.env['decimal.precision'].precision_get('Product Price')
-                if item.currency_id.position == 'after':
-                    item.price = "%s %s" % (
-                        float_repr(
-                            item.fixed_price,
-                            decimal_places,
-                        ),
-                        item.currency_id.symbol,
-                    )
-                else:
-                    item.price = "%s %s" % (
-                        item.currency_id.symbol,
-                        float_repr(
-                            item.fixed_price,
-                            decimal_places,
-                        ),
-                    )
+                item.price = format_amount(self. env,item.fixed_price,item.currency_id)
```

Comment:
```suggestion
                item.price = formatLang(
                    item.env, item.fixed_price, monetary=True, dp="Product Price", currency_obj=item.currency_id)
```
Your bugfix targets lang problems and shouldn't change the rounding applied, which should still be the one of the "Product Price" decimal precision.